index.md

```diff
@@ -57,7 +57,7 @@ Follow the [Getting Started with JanusGraph](http://docs.janusgraph.org/latest/g
 
 JanusGraph is a project under [The Linux
 Foundation](https://www.linux.com/blog/Linux-Foundation-welcomes-JanusGraph),
-and includes participants from Expero, Google, GRAKN.AI, Hortonworks, and IBM.
+and includes participants from Expero, Google, GRAKN.AI, Hortonworks, IBM and Amazon.
```

Comment:
> Perhaps what we should do is call out all orgs by name in the AUTHORS.md list and we update periodically (https://github.com/JanusGraph/janusgraph/blob/master/AUTHORS.txt):
> "and includes participants from Amazon, Aurelius, DataStax, Expero, Google, Orion Health, IBM and other individual authors."
> I think the list of individuals has the potential to get unmanageably large in the future, but the list of organizations will probably be a reasonable size.

The issue is that some of these are copyright holders from the past (via Titan), but strictly speaking, they're not participants in the JanusGraph, so we cannot list them here as that would be a misrepresentation.

Maybe we should just remove the list entirely and link to the AUTHORS list instead? Or maybe list just the companies which are the current developers (vs. copyright holders)?

> BTW why isnt GRAKN.AI and Hortonworks in the AUTHORS.md list? I wonder what the best thing to do is here.

GRAKN.AI and Hortonworks have signed the CCLA but haven't yet contributed code to the repo (unless I missed it), hence they're not in AUTHORS yet.